docs/content/integrations.md

```diff
@@ -33,6 +32,53 @@ have a valid provider ID (Audible, ITunes or ISBN).
 2. Go to your Ryot user settings and add the correct details as described in the
    [yank](#yank-integrations) section.
 
+### Komga
+
+!!! warning
```

Comment:
This warning can be removed. When we add an importer, then it can be added back.